x-pack/plugins/code/public/components/hover/hover_widget.tsx

```diff
@@ -84,7 +84,7 @@ export class HoverWidget extends React.PureComponent<HoverWidgetProps> {
           {
             codeBlockRenderer: (language: string, value: string) => {
               const code = tokenizeToString(value, language);
-              return `<span style="font-family: ${this.props.fontFamily}">${code}</span>`;
+              return `<span>${code}</span>`;
```

Comment:
Also on line 18, remove prop fontFamily 
and line 129 in content_hover_widget.ts